docs/findings/berkeleydb-compat-open-items.md

```
@@ -1,3 +1,8 @@
+> **Status: SHELVED.** The BerkeleyDB compat path was deprioritised in
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)